### PR TITLE
Icons to 160px

### PR DESCRIPTION
**Pull request by bryanbell**
bumped up the icons size a little in support of the new top site
design.

### CODE REVIEW

**Comment by pdehaan** [comment]
Currently seeing 6 errors on Travis-CI:

```
  1563 passing (462ms)
  6 failing

  1) images/ should every file defined in top_sites.json:
     Error: [thedailybeast-com.png] were defined in top_sites.json but they were not found in the images/ folder
      at Context.it (test/image-sizes.test.js:34:13)

  2) images/ should not have unused images:
     Error: [duckduckgo-com.png] were found in the images/ folder but they were not defined in top_sites.json
      at Context.it (test/image-sizes.test.js:40:13)

  3) images/ should have an image with the correct width and height for blackboard:
     AssertionError: height: expected 159 to equal 160
      at Context.it (test/image-sizes.test.js:47:14)

  4) images/ should have an image with the correct width and height for blogger:
     AssertionError: width: expected 159 to equal 160
      at Context.it (test/image-sizes.test.js:46:14)

  5) images/ should have an image with the correct width and height for cnet:
     AssertionError: width: expected 159 to equal 160
      at Context.it (test/image-sizes.test.js:46:14)

  6) images/ should have an image with the correct width and height for thedailybeast:
     Error: ENOENT: no such file or directory, open '/home/travis/build/mozilla/tippy-top-sites/images/thedailybeast-com.png'
      at Error (native)
      at Object.fs.openSync (fs.js:640:18)
      at syncFileToBuffer (node_modules/image-size/lib/index.js:57:23)
      at module.exports (node_modules/image-size/lib/index.js:99:18)
      at Context.it (test/image-sizes.test.js:45:21)
```

---

**TL;DR:**

- ~~Looks like we may have accidentally deleted "images/thedailybeast-com.png".~~
- ~~Have an unused "images/duckduckgo-com.png".~~
- Three missized images (160x159 or 159x160)
    ```sh
    $ node lint-sizes.js
    160x159: images/blackboard-com.png
    159x160: images/blogger-com.png
    159x160: images/cnet-com.png
    ```

**Comment by pdehaan** [comment]
Hey @bryanbell,

It looks like the images for **"blackboard-com.png"**, **"blogger-com.png"**, and **"cnet-com.png"** are off by 1px. Any chance we can get new square versions of those?

**Comment by bryanbell** [comment]
Pushing a fix that makes sure every image is 160x160... and compressed.

**Comment by pdehaan** [comment]
Looks good to me! 👍 
I'll let @k88hudson take a look if she wants, or merge it tomorrow, whatever comes first.

**Comment by rlr** [comment]
Sweet. Thank you @bryanbell and @pdehaan! I'll merge and see if I can remember how to do a release.
